rascal2/ui/view.py

```diff
@@ -69,3 +82,22 @@ def createStatusBar(self):
         """Creates the status bar"""
         sb = QtWidgets.QStatusBar()
         self.setStatusBar(sb)
+
+    def setupMDI(self):
+        """Creates the multi-document interface"""
+        widgets = {
+            "Plots": self.plotting_widget,
+            "Project": self.project_widget,
+            "Terminal": self.terminal_widget,
+            "Fitting Controls": self.controls_widget,
+        }
+
+        for title, widget in reversed(widgets.items()):
+            widget.setWindowTitle(title)
+            window = self.mdi.addSubWindow(
+                widget, QtCore.Qt.WindowType.WindowMinMaxButtonsHint | QtCore.Qt.WindowType.WindowTitleHint
+            )
+            window.setWindowTitle(title)
+        # TODO implement user save for layouts, this should default to use saved layout and fallback to tile
+        self.mdi.tileSubWindows()
```

Comment:
not worth doing if it's particularly difficult (only makes an aesthetic change) but in RasCAL-1, the Tile windows button doesn't do fully symmetric tiling - the terminal and controls widgets are shorter than the plotting and controls. maybe relatively easy with some use of `self.geometry()` to get fractions of the window sizing?

may be outside the scope of this PR - if it is just add it to the TODO comment there and create an issue

a custom tiling function would also make it trivial to implement the user save i think - since we're already setting up custom geometries/sizes for the windows!

Reply:
I don't think the non-symmetric tiling of RasCAL-1 is something we should replicate, it doesn't look good and in my opinion provides no extra functionality. We'll only implement if it is specifically requested, any user that likes that look can drag their windows around and save them.

As for the  height of the controls and the terminal windows, I will wait for both to be implemented before adjusting the window heights.